### ISSUE TITLE
CPU spikes occuring pretty frequently that cause the world to pause then continue what it was doing.  I can still interact with the world during these pauses.

### ISSUE THREAD

**Issue by fumblerful** [ldtteam/minecolonies#2825]
### Minecolonies version

minecolonies-1.12.2-0.9.2-RELEASE-universal.jar

### Expected behavior

- CPU functioning pretty much normally with some heavy load.

### Actual behaviour

- Every few minutes my CPU usage spikes to like 60-80% if not higher.  During these spikes, the game world freezes but I can still move and destroy blocks.  Once the world unfreezes, I am pulled back to  position.  Sometimes my actions while frozen are still done.  I cannot craft during these times.

### Steps to reproduce the problem

1. Have an i7 3820 CPU.  Have Yogbox 2.0 but update minecolonies to the stated version.
2. Start a colony
3. Feel the chill of freezing and watch as task manager shows sharp spikes of cpu usage.

NOTE:  I am unsure if starting the colony is what caused this.  What happened was I found a place to settle after two hours, closed the game, updated minecolonies, and created my colony.  It was at this point that I noticed things going wonky.

**Comment by marchermans** [comment]
Could you use something like WarmRoast, or LagGoggles to do some more debugging. Cause just saying: I play a big pack with lods of mods causes 1 particular mod to cause lag, is not going to get you any help at all.

So please either specify how you deduced that Minecolonies is the issue? Or even better get us WarmRoast and LagGoggles reports.

**Comment by fumblerful** [comment]
So, I made my colony on an island surrounded by ocean.  I had some dungeon towers near by so I killed the boss.  laggoggles then revealed some ents from mo creatures to be causing some issue so I killed them.  However, I am still getting the problem.  I don't think I had issues until trying to found the colony after I installed the mod.  Before, I was cruising around with an older, non working version of minecolonies.  while I could not make a colony, I don't think I was CPU spiking.  After installing minecolonies, I began spiking.  Big emphasis on my game was fine before I updated my version of minecolonies.  I have reduced potential lagging and while that made the lag a better, it was not perfect. I could see additional offenders on some animals around and killed them, still seeing a red line on my unemployed citizen with "25s/t" on his head connected to the line.
